db/schema.rb

```diff
@@ -11,21 +11,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190909093455) do
+ActiveRecord::Schema.define(version: 20190805152649) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "ad_renewal_letters_exports", force: :cascade do |t|
-    t.date     "expires_on"
-    t.string   "file_name"
-    t.integer  "number_of_letters"
-    t.string   "printed_by"
-    t.date     "printed_on"
-    t.datetime "created_at",        null: false
-    t.datetime "updated_at",        null: false
-  end
```

Comment:
WARNING: Do not merge this :D 

Can you remove this file change please? :)

Reply:
Yikes, not sure how that got changed? 😬